1a.cpp

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <vector> 
+using namespace std;
+
+int main()
+{
+	vector<int> v1(5);
+	cout <<"Размер вектора: " <<v1.size()<<"\n";
+	cout <<"Коллекция чисел: \n";
+	int i;
+	for(i=4; i>=0; i--)
+	{
+		int c=0;
+		v1[i]=i*2;		
+		cout<<c+1<<"е число "<<v1[i]<<"\n";
+		c=c+1;
+	}
+	cout <<"Сортировка\n";
+	for (int j=0;j<v1.size();j++)
+	{
+		int posit=j;
+		int temp=v1[posit];
+		for (int k=j+1;k<v1.size();k++)
```

Comment:
`std::vector::size` returns variable of type `std::size_t` which is of unsigned type. `int` is a signed type. By using it, you effectively reduced the number of possible iterations.